File: module_common/src/main/java/module/common/data/request/GiftsReq.java
package module.common.data.request;

import module.common.base.BaseListReq;

public class GiftsReq extends BaseListReq<GiftsReq.QueryObj> {



    public static class QueryObj{

        /**
         * giftTypeId : 1
         */

        private int giftTypeId;

        public QueryObj(int giftTypeId) {
            this.giftTypeId = giftTypeId;
        }

        public int getGiftTypeId() {
            return giftTypeId;
        }

        public void setGiftTypeId(int giftTypeId) {
            this.giftTypeId = giftTypeId;
        }
    }
}

File: module_common/src/main/java/module/common/data/request/GiftsReq.kt
package module.common.data.request

import module.common.base.BaseListReq

class GiftsReq: BaseListReq<GiftsReq.QueryObj>(QueryObj()) {
    class QueryObj {
        /**
         * giftTypeId : 1
         */
        var giftTypeId = 0
    }
}